### ISSUE TITLE
hi sandeep

### ISSUE THREAD

**Issue by ghost** [ManiacSandy/MyRandomSRC#1]
are you the sandeep that I met yesterday at the linuxchix meetup?

:)

github has no messaging feature so im making this issue, ill delete it once we've set up a way to communicate

**Comment by sandeepsanu5** [comment]
Yes... Good to see you again
We could chat over meetup as it has inbuilt messaging.

**Comment by ghost** [comment]
:D

Good to see you again too. It was great chatting with you at the meetup, hope to chat with you again soon.

I don't have the meetup app on my phone. Would you like to try a secure, open-source messenger?
I use an app called Wire - https://github.com/wireapp https://wire.com/

It is available on Android and iOS, my username on wire is `@manas `

**Comment by sandeepsanu5** [comment]
WARNING : Mild Offense ahead
TL;DR : I'm not currently into it, alternatives in consideration

I would like to ( also suggested by few friends of mine ) but i'm tired of using niche application like this ( past experience with telegram and signal ) as there is few interaction other than few who also use the "W" App.

However, there is one application i'm looking forward to and is secure in the sense that it uses PGP Encryption for communication : https://keybase.io

We could do normal email conversation or meetup messaging if it is ok..
Please do suggest if there is a better way for communication

**Comment by ghost** [comment]
No worries, I understand your point of view :)
There is unfortunately a steep curve in trying to get people to shift away from a major app like WhatsApp and towards FOSS alternatives like Wire.

PGP is a great idea! I will make a key and share it with you, I haven't made one for my new `.in` email address yet.

**Comment by sandeepsanu5** [comment]
Keybase is right now invite only, so i could send you an invite if you want one :)
